src/frameworks.sources

```diff
@@ -2029,6 +2029,7 @@ WATCHOS_FRAMEWORKS =        \
 	PassKit                 \
 	PushKit			\
 	Security                \
+	StoreKit                \
```

Comment:
Looking at the complete file, StoreKit is present in all platforms, Mac, iOS, tvOS and watchOS. At this point will should remove it f rom all the subsets and add it to COMMON_FRAMEWORKS